src/pages/docs/chat/rooms/media-sharing.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: "Sharing media with Ably Chat"
+meta_description: "How to share media like images, videos, or files with Ably Chat"
+meta_keywords: "image, file, media, sharing, Ably Chat, chat SDK, realtime messaging, dependability, cost optimisation"
+---
+
+A common requirement for chat applications is the ability to share media like images, videos, or files with other participants.
+
+Ably Chat currently does not support media sharing out of the box, but it is very straightforward to link media and metadata to messages, and this can be used to implement media sharing features.
```

Comment:
This starts out a bit negative. Do we really need to state that we don't have a dedicated API for it?

Reply:
Changed it to a more positive tone, not mentioning we don't support it